plugins/module_utils/io.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c), Yanis Guenane <yanis+ansible@guenane.org>, 2016
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+import os
+import tempfile
+
+
+# This is taken from community.crypto
```

Comment:
May we leverage that collection by adding it as a requirement?

Reply:
We could do that, but in general I would really try to avoid collection dependencies if they are not absolutely required. So far I mainly saw them causing pain in other places :)

Sooner or later it probably makes sense to move code like this to a "proper" dependency,  but right now I think it's too early.

Reply:
(Also it is easy to change this later on.)

Reply:
:+1: good for me then!